docs/qppe.yaml

```
@@ -563,10 +601,22 @@ components:
           type: string
           nullable: true
           description: General feedback that is shown to everyone after scoring.
-      required: [ question_state, num_variants, num_subquestions, score_min,
+      required: [ num_variants, num_subquestions, score_min,
                   score_max, scoring_method, penalty, random_guess_score, subquestions,
                   response_analysis_by_variant, render_every_view, general_feedback ]
 
+    QuestionCreated:
+      allOf:
+        - $ref: '#/components/schemas/Question'
+        - type: object
+          properties:
+            question_state:
+              type: string
+              description: Arbitrary values set by the question package code (LMS must store this data unmodified)
+          required: [ question_state, num_variants, num_subquestions, score_min,
+                      score_max, scoring_method, penalty, random_guess_score, subquestions,
+                      response_analysis_by_variant, render_every_view, general_feedback ]
```

```suggestion
          required: [ question_state ]
```

Dass die anderen required sind, wird durch das `allOf` abgedeckt